ctapipe/image/reducer.py

```diff
@@ -131,6 +118,36 @@ class TailCutsDataVolumeReducer(DataVolumeReducer):
         "normal TailcutCleaning is used.",
     ).tag(config=True)
 
+    def __init__(
```

Comment:
This should have the same signature as the base class to make it configurable from outside the of scripts through the config system.

So I'd suggest going the same route as with the gain selector. So create a traitlet `image_extractor_type` and then use `ImageExtractor.from_name` to create it in the `__init__`

Reply:
The image_extractor type is now not shown in the docs for `TailCutsDataVolumeReducer`, is it normal? Don't know why.

Reply:
What do you mean with "in the docs"?

Reply:
in the docs/_build/html/api after make docs

Reply:
You need to add a docstring to the class with a section `Attributes` for that.